Functions/xASL_bids_PrepareDRO.m

```diff
@@ -0,0 +1,96 @@
+function xASL_bids_PrepareDRO(droTestPatient,droSubject)
+%xASL_bids_PrepareDRO Prepare DRO test patient for BIDS2Legacy conversion.
+%
+% FORMAT: xASL_bids_PrepareDRO(droTestPatient,droSubject)
```

Comment:
OPTIONAL parameters should be in [] here

Reply:
Oh, sorry, I forgot that. Fixed in commit 12959c4.